domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/attendance/Student.kt

```diff
@@ -0,0 +1,36 @@
+package com.krzykrucz.elesson.currentlesson.domain.attendance
+
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.FirstName
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.NumberInRegister
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.SecondName
+
+sealed class Student(val firstName: FirstName,
+                     val secondName: SecondName,
+                     val numberInRegister: NumberInRegister) {
+
+    data class UncheckedStudent(val first: FirstName, val second: SecondName, val no: NumberInRegister) : Student(
```

Comment:
please declare subclasses below the sealed class (not inside it), to make it readable in the place it's used

Reply:
Ok